test/game-test.js

```diff
@@ -56,17 +56,17 @@ describe('Game', function() {
     expect(game.createNewWheel(data)).to.be.an.instanceOf(Wheel);
   });
 
-  it('should display puzzle', () => {
-    domUpdates.startGame(game);
-    expect(domUpdates.startGame).to.have.been.called(1);
+  it.only('should display puzzle', () => {
+    game.startGame()
+    expect(domUpdates.showPuzzle).to.be.spy;
   });
```

Comment:
I have also tried to expect(domUpdates.showPuzzle).to.have.been.called(1) with the same result